common/src/main/java/com/dfsek/terra/config/loaders/palette/slant/SlantHolderLoader.java

```diff
@@ -0,0 +1,30 @@
+package com.dfsek.terra.config.loaders.palette.slant;
+
+import com.dfsek.tectonic.exception.LoadException;
+import com.dfsek.tectonic.loading.ConfigLoader;
+import com.dfsek.tectonic.loading.TypeLoader;
+import com.dfsek.terra.api.world.palette.holder.PaletteHolder;
+import com.dfsek.terra.api.world.palette.slant.SlantHolder;
+
+import java.lang.reflect.Type;
+import java.util.List;
+import java.util.Map;
+import java.util.TreeMap;
+
+@SuppressWarnings("unchecked")
+public class SlantHolderLoader implements TypeLoader<SlantHolder> {
```

Comment:
This should probably be an `ObjectTemplate` instead. If you would like, I can make that change myself later, since I am going to be converting most things to object templates to make the auto-docs more helpful.

Reply:
Would be better for you to handle that :P